docs/developer-docs/latest/update-migration-guides/migration-guides.md

```diff
@@ -5,22 +5,21 @@ description: All the migration guides for a Strapi application.
 
 # Migrations guides
 
-Please also refer to the following [update guide](update-version.md) for a better understanding of how to update your project.
-
-**Pay special attention to the [note](update-version.md#extensions) about upgrading if you are using the extensions system.**
-
 ## Instructions
 
-When upgrading you will need to follow every migration guide between your current version and the one you are upgrading to.
+When upgrading you application you will need to follow every migration guide between your current version and the one you are upgrading to. If there is no specific migration guide for your version step(s), you should follow the general [Update Strapi guide](update-version.md).
```

Comment:
```diff
- When upgrading you application you
+ When upgrading your application you
```

Reply:
`for your version step(s)` I'm not sure to fully understand there.

Reply:
@lauriejim steps between versions. Our migration guides stop at v3.4.4 and we are now 3.6.1 but if we end up needing a migration guide for 3.6.x to say 3.7.x; a "step" for a user here would be 3.4.4 => 3.6.1 => 3.7.x